lib/gauge.js

```diff
@@ -146,14 +146,16 @@ function startTimer(startLabels) {
 function dec(labels) {
 	return value => {
 		const data = convertLabelsAndValues(labels, value);
-		this.set(data.labels, this._getValue(data.labels) - (data.value || 1));
+		const value = data.value === undefined ? 1 : data.value
```

Comment:
Need a different variable name since `value` is also a parameter name.
Also need a semicolon at the end.

(ditto for changes below)